components/blockquote.js

```diff
@@ -0,0 +1,29 @@
+import { Box } from 'theme-ui'
+
+const Blockquote = ({ children }) => {
+  const firstChar = children.props.children.slice(0, 1)
+  const textIndent = firstChar === '“' ? ['-0.4em'] : [0]
```

Comment:
nit: should we also check against `'"'`?

Reply:
haha welllll ideally we shouldn't use that to indicate the start of a quotation (and it'll have an ever so slightly different width), so maybe no?